operator/v1/types_ingress.go

```diff
@@ -536,6 +536,43 @@ type HostNetworkStrategy struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	Protocol IngressControllerProtocol `json:"protocol,omitempty"`
+
+	// HTTPPort is the port on the host which should be used to listen for
+	// HTTP requests. This field should be set when port 80 is already in use.
+	// The value should not coincide with the NodePort range of the cluster.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Maximum=65535
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:default=80
+	HTTPPort int32 `json:"HTTPPort"`
```

Comment:
Shouldn't we stick with the lower camel case convention for the JSON tags?

Reply:
You're right. It looks like all the fields are in lower case even for words like HTTP.